packages/adapters-library/src/adapters/prices-v2/products/usd/pricesV2UsdAdapter.ts

```diff
@@ -129,18 +133,21 @@ export class PricesV2UsdAdapter implements IPricesAdapter {
         },
       })
 
-      if (!response.ok) {
-        throw new Error(`Error fetching spot price: ${response.statusText}`)
+      if (!response.ok || response.status === 204) {
```

Comment:
204  (No Content) is an ok response, but an empty body, which makes response.json() to throw. This treats it as an error.